lib/jgi_mg_assembly/runner/pipeline.py

```diff
@@ -5,20 +5,19 @@
 from jgi_mg_assembly.utils.file import FileUtil
 from jgi_mg_assembly.utils.report import ReportUtil
 from jgi_mg_assembly.utils.util import mkdir
-from jgi_mg_assembly.pipeline_steps.readlength import readlength
+from jgi_mg_assembly.pipeline_steps.readlength import ReadLengthRunner
 from jgi_mg_assembly.pipeline_steps.rqcfilter import RQCFilterRunner
+from jgi_mg_assembly.pipeline_steps.bfc import BFCRunner
+from jgi_mg_assembly.pipeline_steps.seqtk import SeqtkRunner
 from jgi_mg_assembly.pipeline_steps.spades import SpadesRunner
+from jgi_mg_assembly.pipeline_steps.agp import AgpRunner
+from jgi_mg_assembly.pipeline_steps.assemblystats import StatsRunner
+from jgi_mg_assembly.pipeline_steps.bbmap import BBMapRunner
 
-BFC = "/kb/module/bin/bfc"
-SEQTK = "/kb/module/bin/seqtk"
-BBTOOLS_STATS = "/kb/module/bbmap/stats.sh"
-BBMAP = "/kb/module/bbmap/bbmap.sh"
 PIGZ = "pigz"
-AGP_FILE_TOOL = "/kb/module/bbmap/fungalrelease.sh"
 
 
 class Pipeline(object):
-
     def __init__(self, callback_url, scratch_dir):
         """
         Initialize a few things. Starting points, paths, etc.
```

Comment:
Why's the timestamp here set to 1000 seconds in the future?

Reply:
I can't see what line you're referencing, because Github is being obnoxious. the `time.time()*1000` just removes the decimal part of that, so it can be better used as a directory. Really, it's just there to serve as a unique output directory.

Reply:
Ah my bad, I saw a `+` instead of a `*`. multiplying 1000 makes more sense